docs/user/installation_and_configuration.rst

```diff
@@ -0,0 +1,368 @@
+Installation and Configuration
+==============================
+
+Prerequisites and Platform
+--------------------------
+
+.. warning::
+
+    GeoMesa requires Accumulo (http://accumulo.apache.org/) |accumulo_version|, which in turn
+    requires Hadoop (http://hadoop.apache.org/) |hadoop_version| and ZooKeeper (http://zookeeper.apache.org) 
+    |zookeeper_version|. Installing and configuring Accumulo is beyond the scope of this manual.
+
+    Using the Kafka (http://kafka.apache.org/) module requires Kafka |kafka_version| and ZooKeeper |zookeeper_version|.  
+
+To install the binary distribution:
+
+* Java JRE or JDK 7
+
+To build and install the source distribution:
+
+* Java JDK 7
+* Apache Maven (http://maven.apache.org/) |maven_version|
+* A ``git`` client (http://git-scm.com/)
+
+Versions and Downloads
+----------------------
+
+.. note::
+
+    The current recommended version is |release|.
+
+**Latest release**: |release|
+
+.. TODO: substitutions don't work in some kinds of markup, including URLs
+
+* Release tarball: |release_tarball|
+* Source: |release_source_tarball|
+
+**Development version (source only)**: |development|
+
+* Source: https://github.com/locationtech/geomesa/archive/master.tar.gz
```

Comment:
in the test page this shows up as latest release 1.2.0 and development release 1.2.0-SNAPSHOT - presumably that would be 1.3.0-SNAPSHOT?